apollo-router/src/plugins/cache/entity.rs

```diff
@@ -62,6 +63,7 @@ pub(crate) const ENTITY_CACHE_VERSION: &str = "1.0";
 pub(crate) const ENTITIES: &str = "_entities";
 pub(crate) const REPRESENTATIONS: &str = "representations";
 pub(crate) const CONTEXT_CACHE_KEY: &str = "apollo_entity_cache::key";
+pub(crate) const CONTEXT_CACHE_KEYS: &str = "apollo::entity_cache::keys";
```

Comment:
there will be confusion between `apollo_entity_cache::key` and `apollo_entity_cache::keys`. Maybe we should have another name